app/views/curriculum_vitaes/new.html.erb

```diff
@@ -0,0 +1,41 @@
+<% provide:title, t(".title") %>
+<h1><%= t ".title" %></h1>
+
+<div class="row justify-content-md-center">
+  <div class="col-md-6">
+    <%= form_with model: @curriculum_vitae, local: true do |f| %>
+      <%= render "shared/error_messages", object: f.object %>
+
+      <%= f.label :industry, t(".industry") %>
```

Comment:
dùng i18n cho active record chứ không i18n kiểu nàu nhé em 
https://guides.rubyonrails.org/i18n.html#translations-for-active-record-models